project3/style.css

```
@@ -0,0 +1,2586 @@
+/* ===================
+COLOR PALETTE
+===================*/
+
+/* .sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+} */
+
+/* .button-reset[disabled] {
+    cursor: default;
+} */
+
+
+:root {
+    /*  ==================
+  COLOR PALETTE
+  ==================*/
+
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1e2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #fafafa;
+    --color-blue: #163346;
+    /* Reminder that you can name your variables anything you want, they just have to start with two dashes (--) */
+    /* ㅇㅣ거 하면 이름대로 색 바꿀 수 있음 */
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+.button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+button[type="submit"] {
+    background-color: var(--color-green);
+    font-family: Teko;
+    font-weight: 600;
+    font-size: 36px;
+    text-transform: uppercase;
+    margin: auto;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 20px;
+    padding-bottom: 18px;
+    margin-left: 36px;
+    margin-bottom: 100px;
+    margin-top: 40px;
+    border: 5px solid var(--color-green);
+}
+
+button:hover {
+    background-color: var(--color-offwhite);
+    border: 5px solid var(--color-green);
+}
+
+.content {
+    background-color: var(--color-offblack);
+    width: 327px;
+    height: auto;
+    margin: auto;
+    padding-left: 16px;
+    padding-right: 16px;
+    position: relative;
+}
+
+.box {
+    background-color: var(--color-blue);
+    display: inline-block;
+    width: 375px;
+    height: 768px;
+    padding-bottom: 1020%;
+    padding-right: 100%;
+    position: absolute;
+    margin-top: 80px;
+    left: 0;
+}
+
+svg {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.shield {
+    height: 239px;
+    width: auto;
+    /* margin: 68px, 20px, 68px, 40px; */
+    margin: auto;
+    margin-top: 80px;
+}
+
+.strength {
+    height: 101px;
+    width: auto;
+    margin-top: 28px;
+}
+
+.speed {
+    height: 100px;
+    width: auto;
+    margin-top: 48px;
+}
+
+.intelligence {
+    height: 100px;
+    width: auto;
+    margin-top: 48px;
+}
+
+body {
+    background-color: var(--color-grey);
+    color: white;
+    font-family: BenchNine;
```

This is missing the generic font family:
```suggestion
    font-family: BenchNine, sans-serif;
```